app/components/theme.js

```diff
@@ -12,6 +12,10 @@ const Theme = {
     textPrimary: '#000000',
     textBrown: '#403105',
     encivYellow: '#FFC315',
+    encivGray: '#5D5D5C',
```

Comment:
In the next dev meeting I want to talk about theme naming again. It's going to be a recurring group discussion I think, as we figure this out.  And I know someone else already started the pattern but I have two thoughts:
1) EnCiv shouldn't be in the property name - this is all under the enciv repo so it's not useful info.  (Exception  is the two EnCiv logo colors).
2) A color name should not be in the property name.  (exception encivYellow because that the logo color. But maybe we should call it encivLogoYellow.  Instead, and what's harder, is then name should say something about where or why to use it. 
Anyway, it's not required before the group discussion, but see if you can think of other names.